### DISCUSSION TITLE
Scrolling from android is very problematic (the ability to change the shortcuts of the touchpad input gestures )

### DISCUSSION THREAD

**Discussion by hexman1999** [rustdesk/rustdesk#7277] [Feature Request]
### Bug Description

Okay the problem consists of two parts the first one is that scrolling with three fingers already is assigned for screenshot in my Android device which is xiaomi redmi Note 10 Pro so I have to disable the screenshot function from the settings in order to have the scrolling  work and then re-enable it after I finish the other problem appears here which is the very fast and inequivalent ratio for scrolling which is  annoying

### How to Reproduce

My suggestion is to add the ability to change the shortcuts of the touchpad input gestures to the user's liking and adding an option for selecting the speed of the mouse

Best wishes for this promising application 🙂

### Expected Behavior

Scrolling normally

### Operating system(s) on local side and remote side

Android -> windows 10

### RustDesk Version(s) on local side and remote side

1.2.3-> 1.2.3

### Screenshots

![Screenshot_2024-02-27-19-05-19-753_com miui miinput](https://github.com/rustdesk/rustdesk/assets/19696978/98efe5ef-3d13-49d5-999f-88fb2a851891)

**Comment by xlionjuan** [comment]
Second, **lot's of** Android devices' system gesture will conflict to RustDesk's three fingers scroll, so the scroll will not work.

AnyDeak is using three fingers, and user complaints, but they ignored.

TeamViewer is using two fingers.

**Reply by hexman1999** [reply]
I wish if mouse settings can be manually configured I think it'll be much better

**Comment by rustdesk** [comment]
This is a good example if you want a feature request get high priority.
https://github.com/rustdesk/rustdesk/discussions/7437

**Comment by XDM-Inc** [comment]
I'm also having a hard time adjusting to the way that this app works. three finger screenshot has to be turned off every time I want to use this app. It would be nice to have the option to change three finger scrolling to two finger scrolling and lock the panning of the entire desktop to focus in automatically instead of manually

**Reply by nal54** [reply]
I was thinking of the same thing. If they can make this changes then I can uninstall TeamViewer and use this all the time.

**Comment by jax-tr** [comment]
This is also problematic on iOS. Three finger scroll is unreliable and hides too much of the screen while using. Most other apps have 2 finger swipe for scroll, 2 finger pinch for zoom, and 2 finger tap for right click without issue.
At least we should be able to customize inputs. There are so many different uses to RustDesk that it should be as flexible as possible imo.

**Reply by 8BiTw0LF** [reply]
Yup, that's also how splashtop works. Weird this hasn't been corrected in rustdesk yet. I'm using a OnePlus Open and 3-finger swipe doesn't work at all, even after turning off screenshot function.

**Reply by hexman1999** [reply]
> Yup, that's also how splashtop works. Weird this hasn't been corrected in rustdesk yet. I'm using a OnePlus Open and 3-finger swipe doesn't work at all, even after turning off screenshot function.

this is because you have another three finger swipe up gesture for split screen or ai assistant recently

**Comment by MAAB-FW** [comment]
Same problem facing...

**Comment by hexman1999** [comment]
this is a critical thing for mobile users and i wish if it can be implemented nearly

most mobile companies are making three finger swipe for another action already

**Comment by secretlycarl** [comment]
+1, very annoying to have to use 3 fingers. I only have 2 thumbs lol. This is my biggest pain point coming from chrome remote desktop.

I don't have the 3 finger screenshot conflict like others here but the ability to customize the inputs further would be great.

**Comment by hunter99507** [comment]
+1

**Comment by pizzadox11** [comment]
Still noting done 2 years later??
